ios general/ios general/CoreData/OrganizationMO+Custom.m

```diff
@@ -0,0 +1,54 @@
+#import "OrganizationMO+Custom.h"
+#import "EmployeeMO+Custom.h"
+
+@implementation OrganizationMO (Custom)
+
+- (int)calculateAverageSalary
+{
+    NSNumber *salary = [self.employees valueForKeyPath:@"@avg.salary"];
+    int result = [salary intValue];
+
+    for (EmployeeMO *e in self.employees)
+    {
+        NSLog(@"Employee: %@; Salary: %d", e, e.salary);
+    }
+    NSLog(@"Average salary is: %d", result);
+    return result;
+}
+
+- (EmployeeMO *)employeeWithLowestSalary
+{
+    EmployeeMO *employee = [[EmployeeMO alloc] init];
```

Comment:
> Do you really need a default value here?

You mean `MAXFLOAT` could be used as default value here if employee's salary bigger than that?